src/main/scala/com/gu/salesfoce/messageHandler/MessageHandlerLambda.scala

```diff
@@ -0,0 +1,111 @@
+package com.gu.salesfoce.messageHandler
+
+import java.io.{ByteArrayInputStream, InputStream, OutputStream}
+import java.util.concurrent.Executors
+import javax.xml.bind.JAXBContext
+import javax.xml.soap.MessageFactory
+
+import com.amazonaws.services.lambda.runtime.Context
+import com.amazonaws.services.sqs.model.SendMessageResult
+import com.gu.salesfoce.messageHandler.APIGatewayResponse._
+import com.gu.salesfoce.messageHandler.ResponseModels.{ApiResponse, Headers}
+import com.sforce.soap._2005._09.outbound._
+import play.api.libs.json.{JsValue, Json}
+
+import scala.collection.JavaConversions._
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.util.{Failure, Try}
+
+trait RealDependencies {
+  val queueClient = SqsClient
+}
+
+trait MessageHandler extends Logging {
+  def queueClient: QueueClient
+
+  val queueName = s"salesforce-outbound-messages-${Config.stage}"
+  val ThreadCount = 10
+  implicit val executionContext = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(ThreadCount))
+
+  val okXml =
```

Comment:
Also as it's not dynamic, it should probably go on a static object rather than one that's going to be instantiated on every use.